activesupport/lib/active_support/deprecation/reporting.rb

```diff
@@ -36,6 +37,16 @@ def silence
         @silenced = old_silenced
       end
 
+
+      # Outputs a deprecation warning with 'will be removed' message to
+      # the output configured by <tt>ActiveSupport::Deprecation.behavior</tt>.
+      # By default +caller_backtrace+ is equal to caller(2).
+      #
+      #   ActiveSupport::Deprecation.deprecation_warning('something broke!')
```

Comment:
I think offering `'something broke!'` as an example value for a parameter named `deprecated_method_name` is far more confusing than helpful. 

Also, do we have precedent for using `=>` to indicate output instead of a return value?